tox.ini

```diff
@@ -7,16 +7,14 @@ skip_missing_interpreters = True
 envlist = lint, unit
```

Comment:
**todo** I don't think we need these anymore, given we really should not have any python code anymore